src/neptune_scale/logging/logging_handler.py

```diff
@@ -42,11 +42,25 @@
 
 class NeptuneLoggingHandler(logging.Handler):
     """
-    A logging handler that sends log messages to a Neptune run.
-    In its constructor, it takes:
-    - a Neptune run to log to,
-    - a logging level (default is NOTSET),
-    - an optional attribute path under which to store logs (default is "system/logs").
+    A logging handler that sends the records created by the Python Logger to a Neptune run.
+
+    Args:
+        run: A reference to a Run object which should track the logs.
+        level: Level of logs to capture. If not provided, defaults to `logging.NOTSET`.
+        attribute_path: Path to the `StringSeries` attribute that stores the logs. If not provided, 
+            the logs are stored under "system/logs".
```

Comment:
Presumably being changed by #245 

```suggestion
            the logs are stored under "runtime/logs".
```